www/install/sql/centreon/Update-DB-22.10.0-beta.1.sql

```diff
@@ -122,4 +122,6 @@ ALTER TABLE cfg_nagios MODIFY `enable_predictive_service_dependency_checks` enum
 ALTER TABLE cfg_nagios MODIFY `debug_verbosity` enum('0','1');
 
 
-ALTER TABLE contact DROP COLUMN IF EXISTS `enable_one_click_export`;
\ No newline at end of file
+ALTER TABLE contact DROP COLUMN IF EXISTS `enable_one_click_export`;
+
+ALTER TABLE remote_servers DROP COLUMN IF EXISTS `app_key`;
```

Comment:
`DROP COLUMN IF EXISTS` does not work for `MySQL`
```suggestion
ALTER TABLE remote_servers DROP COLUMN `app_key`;
```

Reply:
I will create a ticket to fix query about `enable_one_click_export` (cc @a-launois )